library/system/src/modules/FileChanges.rb

```diff
@@ -231,46 +225,44 @@ def CheckFiles(files)
     def CheckNewCreatedFiles(files)
       new_files = files - @file_checksums.keys
 
-      if new_files.size > 0
-        # Continue/Cancel question, %s is a file name
-        msg = _("File %s has been created manually.\nYaST might lose this file.")
-        if new_files.size > 1
-          # Continue/Cancel question, %s is a comma separated list of file names
-          msg = _(
-            "Files %s have been created manually.\nYaST might lose these files."
-          )
-        end
-        msg = msg % new_files.join(", ")
-        popup_file = "/filechecks_non_verbose"
-        popup_file_path = File.join(Directory.vardir, popup_file)
-        if !FileUtils.Exists(popup_file_path)
-          content = VBox(
-            Label(msg),
-            Left(CheckBox(Id(:disable), Message.DoNotShowMessageAgain())),
-            ButtonBox(
-              PushButton(Id(:ok), Opt(:okButton), Label.ContinueButton()),
-              PushButton(Id(:cancel), Opt(:cancelButton), Label.CancelButton())
-            )
-          )
-          UI.OpenDialog(content)
-          UI.SetFocus(:ok)
-          ret = UI.UserInput
-          Builtins.y2milestone("ret = %1", ret)
-          if ret == :ok && UI.QueryWidget(:disable, :Value)
-            Builtins.y2milestone("Disabled checksum popups")
-            SCR.Write(
-              path(".target.string"),
-              popup_file_path,
-              ""
-            )
-          end
-          UI.CloseDialog
-          return ret == :ok
-        else
-          return true
-        end
+      return true unless new_files.size > 0
+
+      # Continue/Cancel question, %s is a file name
+      msg = _("File %s has been created manually.\nYaST might lose this file.")
+      if new_files.size > 1
+        # Continue/Cancel question, %s is a comma separated list of file names
+        msg = _(
+          "Files %s have been created manually.\nYaST might lose these files."
+        )
+      end
```

Comment:
to be consistent above you use if/else construct, so you can also use it here like 
```
if new_files.size == 1
  ...
else
  ...
end